### ISSUE TITLE
[Prettier] Do not use semicolon

### ISSUE THREAD

**Issue by lesha1201** [datarockets/style-guide#52]
@ezhlobo Proposed to change the default Prettier configuration to stop using semicolons in the code:

```
semi: false
```

https://prettier.io/docs/en/options.html#semicolons

**Comment by ezhlobo** [comment]
@lesha1201 I see we are preparing major upgrade, that seems to be the perfect time to do the change. What do you think?

**Comment by lesha1201** [comment]
@ezhlobo Yes, I think it's a good time. The users of `style-guide` will always be able to easily override the setting anyway.

I'm not sure what is the right process for deciding such things in our case. We can either force certain decisions or we can have a poll among other frontend devs. What do you think? Personally, for me either semi true or false works but I don't want other people to raise similar issue again in the future and would like to come up with some process for deciding such stylistic things. For now, we can just update it since probably it's just me and you who uses `style-guide` but I would like to hear your opinion.

**Comment by ezhlobo** [comment]
@lesha1201 that's a really good question and I'm afraid I don't have answer. Maybe running the poll is the way...
